libraries/bsg_manycore.cpp

```diff
@@ -47,16 +46,6 @@
 #include <climits>
 #include <cstdbool>
 #include <cassert>
-#else
-#include <inttypes>
-#include <stdint.h>
-#include <stdlib.h>
-#include <string.h>
-#include <stdio.h>
-#include <limits.h>
-#include <stdbool.h>
-#include <assert.h>
-#endif
```

Comment:
Seems unnecessary to keep the old c-specific headers, since we have C++ features throughout

Reply:
This has always been 100% unnecessary because this file is C++ source file and will never be compiled without __cplusplus being defined. The only place this maybe makes sense is in header files that might be included by C client source code.